repose-aggregator/commons/utilities/src/test/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapperTest.scala

```diff
@@ -521,6 +523,7 @@ class HttpServletRequestWrapperTest extends FunSpec with BeforeAndAfter with Mat
       wrappedRequest.getPreferredHeader("ornament") shouldBe "santa"
     }
 
+    //todo: fix this test -- shouldn't the higher quality value be returned?
     it("should return the appropriate value when a higher quality is appended onto a line") {
```

Comment:
this one is the squirrely one. the behavior that it should display is not very clear, and is more as we decide it.